tests/skyserve/update/new_autoscaler_before.yaml.j2

```diff
@@ -5,10 +5,10 @@ service:
   replicas: 2
 
 resources:
-  cloud: gcp
+  cloud: {{generic_cloud}}
```

Comment:
Why do we need this templating stuff considering we have the `--cloud` in our test already?

Reply:
Even though the test yaml with `cloud: gcp` worked as we expected due to `--cloud` option, it was confusing and I was not able to come up with a good reason to keep it as `cloud: gcp` while it is being tested for other clouds as well. Should we keep it as it was without templating?

Reply:
Let's keep the original way or just remove the `cloud:` in the YAML, as it seems unnecessary to do a template when our `CLIs` are built for being able to override the fields.

Reply:
Removed the `cloud:` from YAML, and reverted the template :)